### PR TITLE
[MMC-1808] Published internal SDK repo to external repo. Added .gitignore.

### PR DESCRIPTION
**Pull request by michael-patzer**
This clones the internal SDK repo. Once this is merged the source code will be available to the public.

Testing

1. Clone the repo.
2. Attempt to build Panion.
3. Make sure all unit tests pass.
5. Do a quick glance to make sure that nothing is being committed that shouldn't be (.DS_STORE files, personal xcschemes, etc.)

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    FileUtils.copy(license_path, deploy_path)
+
+    # Zip the framework, README, and Changelog
+    zip(path: "deploy", output_path: deploy_zip)
+
+    # Remove temporary deploy directory
+    FileUtils.remove_dir(deploy_path, force = true)
+
+    set_github_release(
+      repository_name: "mailchimp/Mailchimp-SDK-iOS-Internal", # This will change to the non-Internal repo
```
Comment (jennifer-starratt) [inline fastlane/Fastfile]: Should we go ahead and change this?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  end
+
+  desc "Create Swift Binary Framework"
+  lane :create_binary_framework do
+    archive_framework
+
+    # Remove any existing xcframeworks on local machine
+    sh("rm -rf #{xcframework_output_path}")
+
+    # When Xcode 11 is officially released, fastlane may support -create-xcframework in the xcodebuild() command which could replace this (https://github.com/fastlane/fastlane/issues/15055)
```
Comment (jennifer-starratt) [inline fastlane/Fastfile]: Is this still true? I think fastlane has had that issue opened/closed multiple times.

File: MailchimpSDK/MailchimpSDKTests/MailchimpSDKTests.swift [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+                    XCTAssertEqual(response.status, 404)
+                    XCTAssertEqual(response.type, "http://developer.mailchimp.com/documentation/mailchimp/guides/error-glossary/")
+                    XCTAssertEqual(response.detail, "Invalid path")
+                }
+            }
+            testExpectation.fulfill()
+        }
+        XCTAssertNotNil(dataTask)
+        waitForExpectations(timeout: timeout, handler: nil)
+    }
```
Comment (jennifer-starratt) [inline MailchimpSDK/MailchimpSDKTests/MailchimpSDKTests.swift]: Should we keep these commented out tests?

File: fastlane/Fastfile (hunk 2 above)
Comment (michael-patzer) [reply fastlane/Fastfile]: I can't find any indication from fastlane that this is supported yet.

File: MailchimpSDK/MailchimpSDKTests/MailchimpSDKTests.swift (hunk 3 above)
Comment (michael-patzer) [reply MailchimpSDK/MailchimpSDKTests/MailchimpSDKTests.swift]: Removed. Thanks!

File: fastlane/Fastfile (hunk 2 above)
Comment (jennifer-starratt) [reply fastlane/Fastfile]: Same :(

**Comment by jennifer-starratt** [comment]
Updated prior comment with latest changes; still curious if we can remove `MailchimpSDK/MailchimpSDK/Panion/` (did it locally and everything *seemed* okay)

**Pull request merged**